module/proto/gravity/v1/gravity.proto

```diff
@@ -0,0 +1,92 @@
+syntax = "proto3";
+package gravity.v1;
+
+import "cosmos_proto/cosmos.proto";
+import "google/protobuf/timestamp.proto";
+import "gogoproto/gogo.proto";
+import "cosmos/base/v1beta1/coin.proto";
+import "google/protobuf/any.proto";
+
+option go_package = "github.com/cosmos/gravity-bridge/module/x/gravity/types";
+
+// EthereumEventVoteRecord is an event that is pending of confirmation by 2/3 of the signer set.
+// The event is then attested and executed in the state machine once the required
+// threshold is met.
+message EthereumEventVoteRecord {
+  google.protobuf.Any event    = 1 [
+    (cosmos_proto.accepts_interface) = "EthereumEvent"
+  ];  
+  repeated string     votes    = 2;
+  bool                accepted = 3;
+}
+
+// LatestEthereumBlockHeight defines the latest observed ethereum block height and the
+// corresponding timestamp value in nanoseconds.
+message LatestEthereumBlockHeight {
+  uint64 ethereum_height = 1;
+  uint64 cosmos_height   = 2;
```

Comment:
imo this should be a timestamp, unless you are using the height with the epoch/version number (i.e `4` from the chain id `cosmoshub-4`)

Reply:
Yeah interested to hear what the issue would be.